tests/api/notifications/get.test.js

```diff
@@ -1,12 +1,22 @@
 const CONFIG = require('config')
 const __ = CONFIG.universalPath
 require('should')
-const { authReq } = __.require('apiTests', 'utils/utils')
+const { getUser, getReservedUser, authReq } = __.require('apiTests', 'utils/utils')
+const { makeFriends } = __.require('apiTests', 'utils/relations')
 
 describe('notifications:get', () => {
   it('should get user notifications', async () => {
+    const user = await getUser()
+    const friend = await getReservedUser()
+    await makeFriends(user, friend)
+
     const { notifications, total } = await authReq('get', '/api/notifications')
     notifications.should.be.an.Array()
-    total.should.be.aboveOrEqual(0)
+    total.should.be.equal(1)
+
+    const notif = notifications[0]
```

Comment:
the above comment impact this line too, you may want to pop the array to be sure to pick the lasted created notif